src/test/java/mate/academy/security/CustomUserDetailsServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.security;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+class CustomUserDetailsServiceTest {
+    private static final String BOB_EMAIL = "bob@gmail.com";
+    private static final String PASSWORD = "1234";
+    private static final long ID = 1L;
+    private static final Set<Role> ROLES = Set.of(new Role(Role.RoleName.USER));
+    private static final String ALICE_EMAIL = "alice@gmail.com";
+    private UserDetailsService userDetailsService;
+    private UserService userService;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        userService = Mockito.mock(UserService.class);
+        userDetailsService = new CustomUserDetailsService(userService);
```

Comment:
why do we initialize it before every test? I suppose it will be enough to make it only once